src/main/java/org/craftercms/studio/controller/rest/v2/GroupsController.java

```diff
@@ -112,14 +125,19 @@ public ResponseBody createGroup(@RequestBody Group group) throws GroupAlreadyExi
      * @return Response object
      */
     @PatchMapping(value = "/api/2/groups", consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseBody updateGroup(@RequestBody Group group) throws ServiceLayerException {
-        Group updatedGroup = groupService.updateGroup(DEFAULT_ORGANIZATION_ID, group);
-
+    public ResponseBody updateGroup(@RequestBody GroupTO group) throws ServiceLayerException {
         ResponseBody responseBody = new ResponseBody();
-        ResultOne<Group> result = new ResultOne<>();
-        result.setResponse(ApiResponse.OK);
-        result.setEntity(updatedGroup);
-        responseBody.setResult(result);
+        try {
+            GroupTO updatedGroup = groupService.updateGroup(DEFAULT_ORGANIZATION_ID, group);
+
+            ResultOne<GroupTO> result = new ResultOne<>();
+            result.setResponse(ApiResponse.OK);
+            result.setEntity(updatedGroup);
+            responseBody.setResult(result);
+        } catch (GroupNotFoundException e) {
```

Comment:
Can't we move this to ExceptionHandlers? That's where we're currently putting all our API exception handling code.